<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Droper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/droper.svg</normaloff>:/droper.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="currentFolderLabel">
      <property name="text">
       <string>Dropbox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="filesAndFoldersListWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="showAccountInfoAction"/>
    <addaction name="showDownloadAction"/>
    <addaction name="forgetAuthenticationAction"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="showAccountInfoAction">
   <property name="text">
    <string>Show Account Info</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="forgetAuthenticationAction">
   <property name="text">
    <string>Forget Authentication</string>
   </property>
  </action>
  <action name="showDownloadAction">
   <property name="text">
    <string>Show Download</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="renameAction">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="createFolderAction">
   <property name="text">
    <string>Create Folder</string>
   </property>
  </action>
  <action name="upAction">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
  <action name="refreshAction">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
